doc-content/enterprise-only/openshift/customimage-verify-partstep.adoc

```diff
@@ -0,0 +1,14 @@
+. After the build completes, run the image, log into it, and verify that the customization was successful. Enter the following command:
```

Comment:
This looks like two steps, but I think you mean:
After the build completes, enter the following command to run the image...